src/server/DeviceServer.js

```diff
@@ -361,9 +362,16 @@ class DeviceServer {
       );
 
       const description = await device.getDescription();
-      const { uuid } = FirmwareManager.getAppModule(
-        description.systemInformation,
-      );
+      // const { uuid } = FirmwareManager.getAppModule(
+      //   description.systemInformation,
+      // );
+      try {
+        uuid = FirmwareManager.getAppModule(
+          description.systemInformation,
+        )[0];
```

Comment:
at this moment i can't try it with one real Photon.
`const {uuid} uuid = FirmwareManager.getAppModule(
          description.systemInformation,
);` 
is egual to       
`uuid = FirmwareManager.getAppModule(
          description.systemInformation,
)[0];`
I'm wrong?